src/components/Tool/NavToolList/NavToolList.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { ReactElement } from 'react';
+import {
+  makeStyles,
+  Box,
+  Theme,
+  List,
+  ListItem,
+  ListItemText,
+  ListItemIcon,
+} from '@material-ui/core';
+import HomeIcon from '@material-ui/icons/Home';
+import PeopleIcon from '@material-ui/icons/People';
+import CloudUploadIcon from '@material-ui/icons/CloudUpload';
+import BuildIcon from '@material-ui/icons/Build';
+import { useTranslation } from 'react-i18next';
+import { ToolNavItems } from './ToolNavItems';
+import { Item } from './Item/Item';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  list: {
+    width: '290px',
+    transform: 'translateY(55px)',
+    [theme.breakpoints.down('xs')]: {
+      transform: 'translateY(45px)',
+    },
+  },
+  li: {
+    borderTop: '1px solid',
+    borderBottom: '1px solid',
+    borderColor: theme.palette.cruGrayDark.main,
+  },
+}));
+
+const NavToolList = (): ReactElement => {
+  const classes = useStyles();
+  const { t } = useTranslation();
+
+  return (
+    <Box component="div" className={classes.list}>
+      <List>
+        <ListItem className={classes.li}>
+          <ListItemIcon>
+            <BuildIcon />
+          </ListItemIcon>
+          <ListItemText primary={t('Tools')} />
+        </ListItem>
+        {ToolNavItems.slice(0, 1).map((tool) => (
```

Comment:
Since you're parsing your own JSON, could you just group them in the JSON and iterate through the groups? Slicing here with magic numbers seems a little weird and error prone here. It could also make this list component more reusable.
```ts
const ToolsListGroups = [
  {
    groupName: 'Tools',
    items: [
      {
        tool: 'Appeal',
        desc: 'Set goals, create asks, and track progress for one-time needs',
        icon: mdiTrophy,
        id: 'appeal',
      },
      ...
    ],
  },
  ...
];
```

(I wrote this before I discovered you had 2 sets of JSON tools. They could probably still be combined like this. When you don't want them grouped, you could use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flatMap to get rid of the groups.)

Reply:
Grouped the tools like above and made it sharable between the landing page and the side bar. Also removed the other tool list file. The order of the tools on the landing page is now the same as the side bar, but if we want it to be like in the original, I can add some extra code to do it.

Reply:
I don't think the order matters. The sidebar order is probably better anyways.